src/Routes.tsx

```diff
@@ -8,8 +8,10 @@ import { Navigate, ReactLocation, Router } from '@tanstack/react-location';
 
 import { Families, Projects } from './pages';
 import { DashboardLayout } from './pages/Dashboard/DashboardLayout';
+import { Login } from './pages/Login';
 
 const location = new ReactLocation();
+const isAuthorized = false;
```

Comment:
let's just add a route for login (/login) instead of making a pesudo auth

Reply:
Yeah it is a good idea for now.